### ISSUE TITLE
HTML message links are mangled

### ISSUE THREAD

**Issue by tubaman** [phpList/phplist-plugin-restapi#61]
When creating a new campaign via the REST API, HTML message links get mangled with triple backslashes.  The textmessage links are not affected(they work just fine).  I've confirmed this by creating a campaign via the web UI, then creating a similar campaign via the REST API, and comparing the raw data in mysql.

**Comment by michield** [comment]
Yes, I'm aware of this, and it's very annoying. It's kind of outdated handling of the slashes in phpList main, not the Rest API. It needs attention.
